.github/workflows/build.yml

```
@@ -34,6 +34,7 @@ jobs:
     - name: Do docker stuff
       run: ./gradlew doDockerPreWork --no-daemon --parallel --continue
     - uses: docker/build-push-action@v1
+      if: ${{github.event != "pull_request"}}
```

```suggestion
      if: ${{github.event_name != 'pull_request'}}
```
Looks like from here it should be `event_name`

https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsif